### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by vre2h**
I've added a little info about project

### CODE REVIEW

**Comment by hov001** [comment]
Please add my name

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# HTML/CSS
+
+A small project which is going to use following tech stack:
+
+- HTML
+- CSS
+- Google Fonts
+
+Collaborators: @hov001, @vahan11
```
Comment (hov001) [inline README.md]: Please add my full name

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# HTML/CSS
+
+A small project which is going to use following tech stack:
+
+- HTML
+- CSS
```
Comment (Vahan11) [inline README.md:6]: What about flexbox?

File: README.md (hunk 1 above)
Comment (Vahan11) [inline README.md]: What about other guys?

**Review by Vahan11 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vre2h) [reply README.md:6]: Okay, will be done

File: README.md (hunk 1 above)
Comment (hov001) [reply README.md]: Ladno ne nado

**Review by Vahan11 — APPROVED** [review]
Great job @vre2h jan

**Pull request merged**